webapp/src/components/Question.vue

```diff
@@ -50,16 +70,20 @@
 <script>
 import questionService from "@/services/questions.service.js";
 import ModalDoubt from "@/components/ModalDoubt";
+import Solutions from "./Solutions";
```

Comment:
Se for uma solução só, deveria ser no singular o nome desse bagulho.

Reply:
É que em breve teremos mais de uma solução, então achamos melhor deixar como "solutions"